### DISCUSSION TITLE
GitHub Education not accepting my documents.

### DISCUSSION THREAD

**Discussion by sudebozkurt** [community/community#68038] [GitHub Education]
### Hi everyone,

Hi everyone, 
I'm encountering some errors while applying for Github Education. When uploading my student document, it says "The image you uploaded appears to be from a third party." However, I am using the document we need to provide to institutions in Turkey to prove our student status.

Also when I try to upload my school-issued student ID card to the system, I receive an error stating that it's not visible, even though my school name is clearly shown.

Thanks for helps. Have a nice day.

**Comment by tiagoangelos** [comment]
Try scanning your documents using the ClearScaner application, this may solve the readability problem, don't try to contact them and explain, I hope I helped!

app link: https://play.google.com/store/apps/details?id=com.indymobileapp.document.scanner

**Reply by sudebozkurt** [reply]
I tried but it didn't work

**Comment by mm0n2t** [comment]
You can submit a ticket through https://support.github.com/features/education. It will take some time. I got mine after about 1 month.
